sru/modules.py

```diff
@@ -161,13 +163,7 @@ def __init__(self,
         self.reset_parameters()
 
     def reset_parameters(self):
-        """Properly initialize the weights of SRU, following the same
-        recipe as:
-        Xavier init:
-            http://proceedings.mlr.press/v9/glorot10a/glorot10a.pdf
-        Kaiming init:
-            https://arxiv.org/abs/1502.01852
-
+        """Properly initialize the weights of SRU.
```

Comment:
suggest removing `Properly`, since add no new information on what initialization scheme is being used.

Reply:
will do.